### PR TITLE
Add neutral cat, good cat, evil cat, wizard and book gifs

### PR DESCRIPTION
**Pull request by jsahim**
Change summary: Added neutral cat, good cat, good cat flying, evil cat, wizard and book gifs

### CODE REVIEW

File: src/images/good-cat-fly.gif
Comment (BriBourassa) [inline src/images/good-cat-fly.gif:1]: This might be the single cutest thing I've ever seen!! Amazing work Jeff

**Review by BriBourassa — APPROVED** [review]
All of these are so dang cute!! The good cat flying and the Wizard's new robes are incredible

File: src/images/neutral-cat.gif
Comment (qrispi) [inline src/images/neutral-cat.gif:1]: Oh I like how you made sure the images all start at the bottom of the file!

**Review by Jesusborjas006 — APPROVED** [review]
These are amazing!

**Review by qrispi — APPROVED** [review]
Awesome work Jeff! Now we actually have a game coming together!

**Pull request merged**